delivery_readmes/DELIVERY.README.RNASeq.txt

```diff
@@ -0,0 +1,119 @@
+README
+=======
+
+The README describes the content of the delivered data.
+-----------------------------------------------------------------
+
+====================================================================
+ProjectID
+====================================================================
+The root folder, which is named by the NGI project ID, contains one report folder, one analysis results folder and a folder for each sample.
+Each sample folder is accompanied by a .lst-file containing a list of the files in the folder and a .md5-file containing the MD5-checksums of the files in the folder.
+Use the MD5-checksums to verify the integrity of the files after transfer.
+
+├──ProjectID
+	├── 00-Reports
+	├── 01-RNA-Results
+        ├── Sample1
+        ├── Sample1.lst
+        ├── Sample1.md5
+        ├── Sample2
```

Comment:
If a folder, good to suffix with `/` so that it's clear that this isn't a flat file like the rest.

Reply:
Done!